### ISSUE TITLE
@timer not firing

### ISSUE THREAD

**Issue by Soulless-1** [Sphereserver/Source-X#846]
two issues with @timer not triggering properly

1)  if an item is on a memory layer, on a mounted horse, the item will not trigger @timer. This is a problem for things like custom pet hunger systems, and spell effects, and the conjured mounted pets. Also causes issues for custom stable systems that would feed and take care of your pets on a set schedule. I propose that @timer should always fire on something equipped on a char, regardless of being in a sleeping sector, being asleep itself, or being mounted. 

2) items in sleeping sectors that are just on the ground will not trigger @timer. So if someone puts down 25 stacks of fish at the entrance of shame, or someone dies right by the entrance. once a player walks in they will see all this stuff, and then it triggers @timer and will decay. This has been reported countless times on my shard and it just causes confusion for everyone who sees it.

**Comment by Tolokio** [comment]
I think drk already did something to fix this or something similar. Can anyone confirm status of the bug?

At stable, timers will not run, but mounted maunts should run any timer.

**Comment by Tolokio** [comment]
updatE:

1. I did a PR that should fix timers not firing on ridden mounts, but timers should work without the fix unless the mount is harmed.
Plz confirm timers works if mount is not harmed while ridden. Last time I tested there was no problem if mount wasnt harmed. If it happens it should be cos mount is on a sleep sector (the invis mount). Try to confirm this as well.
There is a can to allow items to dont enter in sleep mode but i dont think it works if the timer is equipped on a sleeped sector.

2. 
This can be fixed by script, using @dropon_floor to add the "can" that allow items works on sleeping sectors. And removing the "can" when it is dragged.
Not sure how this should be fixed by code.
